<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MrGaze</class>
 <widget class="QMainWindow" name="MrGaze">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MrGaze</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>781</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Setup_Tab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QGroupBox" name="pupilThresholdGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Pupil Threshold</string>
      </property>
      <widget class="QDial" name="Intensity_LB_Dial">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="notchTarget">
        <double>5.000000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="Intensity_LB_Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="glintThresholdGroup">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Glint Threshold</string>
      </property>
      <widget class="QLabel" name="Intensity_UB_Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QDial" name="Intensity_UB_Dial">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="notchTarget">
        <double>5.000000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="videoSourceGroup">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>551</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Video Source</string>
      </property>
      <widget class="QLabel" name="videoFileLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>File</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QToolButton" name="videoFileTool">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>60</y>
         <width>26</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QComboBox" name="videoSourceCombo">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>141</width>
         <height>26</height>
        </rect>
       </property>
       <property name="currentText">
        <string>USB Camera 0</string>
       </property>
       <item>
        <property name="text">
         <string>USB Camera 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB Camera 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB Camera 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Video File</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="videoSourceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Source</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="videoFileEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>401</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="videoControlsBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Video Control</string>
      </property>
      <widget class="QPushButton" name="playButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/Pause.png</normaloff>
         <normalon>:/icons/Play.png</normalon>:/icons/Pause.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="restartButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/Restart.png</normaloff>
         <normalon>:/icons/Restart.png</normalon>:/icons/Restart.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="rewindButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/Rewind.png</normaloff>
         <normalon>:/icons/Rewind.png</normalon>:/icons/Rewind.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="fastforwardButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/FastForward.png</normaloff>
         <normalon>:/icons/FastForward.png</normalon>:/icons/FastForward.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="toEndButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/ToEnd.png</normaloff>
         <normalon>:/icons/ToEnd.png</normalon>:/icons/ToEnd.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Setup</string>
      </property>
      <widget class="QPushButton" name="saveSetupButton">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Setup</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>USB iSight</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB xiQ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB Thorlabs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>File</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="pupilClassifierLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Pupil Classifier</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Calibrate_Tab">
     <attribute name="title">
      <string>Calibrate</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Gaze_Tab">
     <attribute name="title">
      <string>Gaze</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="Video">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>BLINK</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Gaze">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>BLINK</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Intensity_LB_Dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MrGaze</receiver>
   <slot>update_intensity_lb()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Intensity_UB_Dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MrGaze</receiver>
   <slot>update_intensity_ub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_intensity_lb()</slot>
  <slot>update_intensity_ub()</slot>
 </slots>
</ui>
